config-connector/solutions/sql/kpt/mysql-ha/README.md

```diff
@@ -0,0 +1,32 @@
+MySQL High Availability
+==================================================
+# NAME
+  mysql-ha
+# SYNOPSIS
+  Config Connector compatible YAMLs for creating a high availability MySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/sql/kpt/mysql-ha mysql-ha
+  ```
+# USAGE
+## SETTERS
+|     NAME      |       VALUE       |     SET BY      |       DESCRIPTION       | COUNT |
+|---------------|-------------------|-----------------|-------------------------|-------|
+| database-name | mysql-ha-database | package-default | name of SQL database    | 1     |
+| instance-name | mysql-ha-instance | package-default | name of SQL instance    | 5     |
+| password      | ${PASSWORD?}      | PLACEHOLDER     | password of SQL user    | 1     |
+| region        | us-central1       | package-default | region of SQL instance  | 2     |
+| user-name     | mysql-ha-user     | package-default | name of SQL user        | 1     |
+| zone          | us-central1-b     | package-default | zone of instance        | 1     |
+| zone-backup   | us-central1-c     | package-default | zone of backup instance | 1     |
+
+  Configure setters using kpt as follows:
```

Comment:
We should mention that password is base64 encoded value and reference [how to create secret manually](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually) that explains how to create base64 encoded value.

Reply:
We should also remind users never to check in base64 encoded secrets into source repository, because doing so means that secrets are compromised. We can recommend to refer to [K8s secret management](https://kubernetes.io/docs/concepts/configuration/secret/) for best practices on secret creation. (i.e. no need to go into long guidance, just a short reminder and reference).

Reply:
I have added instructions to base64 encode the password.